<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerpage</class>
 <widget class="QDialog" name="managerpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>811</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="manager_logout">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_showPro">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Products</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_showCATEG">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Category</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_adddispro">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Discount (Pro)</string>
   </property>
  </widget>
  <widget class="QLabel" name="man_hi_lable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hi Manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="myapplogo2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>My App</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_addPro">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_adddiscode">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Discount (Code)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_users">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>USERS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_edditPro">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Products</string>
   </property>
  </widget>
  <widget class="QLabel" name="man_name_managerpage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pm_lable">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>120</y>
     <width>811</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="man_addsalebutton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD AUCTION</string>
   </property>
  </widget>
  <widget class="QLabel" name="digital_time">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>man_addPro</tabstop>
  <tabstop>man_showPro</tabstop>
  <tabstop>man_showCATEG</tabstop>
  <tabstop>man_adddispro</tabstop>
  <tabstop>man_adddiscode</tabstop>
  <tabstop>man_users</tabstop>
  <tabstop>manager_logout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
